s.py

```diff
@@ -0,0 +1,229 @@
+    name: str = devs.get("name")
+    twitter_username: str = devs.get("twitter_username")
+    github_username: str = devs.get("github_username")
+    location: str = devs.get("location")
+    joined_at: Union[str, int] = devs.get("joined_at")
+    profile_url: str = f"https://dev.to/{username}"
+
+    dev_to_be_parsed: Everyone = Everyone(
+        id=id,
+        profile_url=profile_url,
+        username=username,
+        name=name,
+        github_username=github_username,
+        github_url="",
+        twitter_url="",
+        twitter_username=twitter_username,
+        location=location,
+        joined_at=joined_at,
+        linked_in="",
+        email="",
+        work="",
+        stackOverflow="",
+        instagram="",
+        facebook="",
+        youtube="",
+        education="",
+        work_status="",
+    )
+    if username:
+        partial_dev_data.append(dev_to_be_parsed)
+
+    #print(partial_dev_data)
+
+    return partial_dev_data
+
+
+
+def parse_devto_profiles(partial_developers: List[Everyone]):
+    everyone = get_profile_url()
+    for developer in partial_developers:
+        profile_page = requests.get(everyone[0].profile_url)
+        soup = BeautifulSoup(profile_page.text, 'lxml')
+
+        meta = soup.find_all('div',{'class': ['key', 'value']})
+        keys: List[str] = []
+        values: List[str] = []
+        for tag in meta:
+            if tag.get("class") == ["key"]:
+                keys.append(tag.text.strip().replace("\n", ""))
+            elif tag.get("class") == ["value"]:
+                values.append(tag.text.strip().replace("\n", ""))
+        #print(keys)
+        #print(values)
+
+        socials = soup.find_all('a',{'target' : ['_blank']})
+
+        social_info: Dict[str, str] = dict(zip(keys, values))
+
+        links: List[str] = []
+        for a in socials:
+            links.append(a['href'])
+
+        for link_in in links:
+            if link_in.startswith("https://www.linkedin.com"):
+                social_info['linked_in'] = link_in
+
+        for link_git in links:
+            if link_git.startswith('https://github.com'):
+                social_info['github'] = link_git
+
+        for link_twitter in links:
+            if link_twitter.startswith('https://twitter.com'):
+                social_info['twitter'] = link_twitter
+
+        for link_stack in links:
+            if link_stack.startswith('https://stackoverflow.com'):
+                social_info['stack'] = link_stack
+
+        for link_fb in links:
+            if link_fb.startswith('https://www.facebook.com'):
+                social_info['facebook'] = link_fb
+
+        for link_inst in links:
+            if link_inst.startswith('https://www.instagram.com'):
+                social_info['instagram'] = link_inst
+        #print(f"LINKEDIN: {social_info.get('linked_in')}")
+        for link_yt in links:
+            if link_yt.startswith('http://youtube.com'):
+                social_info['youtube'] = link_yt
```

Comment:
You can do this in 1 loop instead of 7 😄 

```suggestion
        for link in links:
            if link.startswith("https://www.linkedin.com"):
                social_info['linked_in'] = link
            elif link.startswith('https://github.com'):
                social_info['github'] = link
            elif link.startswith('https://twitter.com'):
                social_info['twitter'] = link
            elif link.startswith('https://stackoverflow.com'):
                social_info['stack'] = link
            elif link.startswith('https://www.facebook.com'):
                social_info['facebook'] = link
            elif link.startswith('https://www.instagram.com'):
                social_info['instagram'] = link
            elif link.startswith('http://youtube.com'):
                social_info['youtube'] = link
```